camera/camera_balsaq.cc

```diff
@@ -76,7 +79,10 @@ void CameraBq::loop() {
     message.width = camera_frame.size().width;
     message.camera_serial_number = camera_serial_number_;
     publisher_->publish(message);
-    std::cout << "Camera BQ: publishes a camera frame " << message.width << " " << message.height << std::endl;
+
+    std::stringstream go_msg;
```

Comment:
does this just avoid having to explicitly cast things to strings?

Reply:
`std::string` doesn't have a `<<` operator

Reply:
of course it doesn't...